hugo/content/guides/multiple-languages.md

```diff
@@ -0,0 +1,456 @@
+
+Before the change only one language and grammar was defined:
+
+```js
+//...
+"contributes": {
+    "languages": [
+        {
+            "id": "multiple-languages",
+            "aliases": [
+                "Multiple Languages",
+                "multiple-languages"
+            ],
+            "extensions": [".hello"],
+            "configuration": "./language-configuration.json"
+        }
+    ],
+    "grammars": [
+        {
+            "language": "multiple-languages",
+            "scopeName": "source.multiple-languages",
+            "path": "./syntaxes/multiple-languages.tmLanguage.json"
+        }
+    ]
+},
+//...
+```
+
+After the change, we tripled the information. Be aware of that the language ids must match the ids from the Langium configuration. Also make sure that the paths to the syntax files and the language configuration are correct.
+
+> For the language configuration for VSCode, we reused the old file three times. If you want to make a more precise configuration per language, you should also split this file. But let's use the same for a moment and for simplicity.
+
+```js
+//...
+"contributes": {
+    "languages": [
+        {
+            "id": "multiple-languages-configuration",
+            "aliases": [
+                "Multiple Languages Configuration",
+                "multiple-languages-configuration"
+            ],
+            "extensions": [".me"],
+            "configuration": "./language-configuration.json"
+        }, {
+            "id": "multiple-languages-definition",
+            "aliases": [
+                "Multiple Languages Definition",
+                "multiple-languages-definition"
+            ],
+            "extensions": [".who"],
+            "configuration": "./language-configuration.json"
+        }, {
+            "id": "multiple-languages-implementation",
+            "aliases": [
+                "Multiple Languages Implementation",
+                "multiple-languages-implementation"
+            ],
+            "extensions": [".hello"],
+            "configuration": "./language-configuration.json"
+        }
+    ],
+    "grammars": [
+        {
+            "language": "multiple-languages-configuration",
+            "scopeName": "source.multiple-languages-configuration",
+            "path": "./syntaxes/multiple-languages-configuration.tmLanguage.json"
+        },
+        {
+            "language": "multiple-languages-definition",
+            "scopeName": "source.multiple-languages-definition",
+            "path": "./syntaxes/multiple-languages-definition.tmLanguage.json"
+        },
+        {
+            "language": "multiple-languages-implementation",
+            "scopeName": "source.multiple-languages-implementation",
+            "path": "./syntaxes/multiple-languages-implementation.tmLanguage.json"
+        }
+    ]
+},  
+```
+
+#### File `src/extension/main.ts`
+
+And here is the extension file and how the file watcher in it worked before the change:
```

Comment:
After https://github.com/eclipse-langium/langium/pull/1286, we won't need to define any file system watcher in the client. We should also check for other parts of the documentation where this needs to be updated.

Reply:
I removed the part about the file watcher. The remaining parts look ok to me.